chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -223,17 +223,21 @@ def backward_gpu(self, inputs, grad_outputs):
 def softmax_cross_entropy(
         x, t, normalize=True, cache_score=True, class_weight=None,
         ignore_label=-1, reduce='mean'):
-    """Computes cross entropy loss for pre-softmax activations.
+    """Computes cross entropy loss after softmax activations.
```

Comment:
It seems that the previous sentence is better; in your sentence, it is a bit confusing that we should apply "softmax" functions before calling `softmax_cross_entropy`.

Reply:
revereted it.